R/install-remote.R

```diff
@@ -32,11 +32,23 @@ install_remotes <- function(remotes, ...) {
 # Add metadata
 add_metadata <- function(pkg_path, meta) {
   path <- file.path(pkg_path, "DESCRIPTION")
-  desc <- read_dcf(path)
+  if (file.exists(path)) {
+    desc <- read_dcf(path)
+
+    desc <- modifyList(desc, meta)
+
+    write_dcf(path, desc)
+  }
 
-  desc <- modifyList(desc, meta)
+  path <- file.path(pkg_path, "Meta", "package.rds")
+  if (file.exists(path)) {
```

Comment:
I think it should be an error if neither can be found, but agree it is a good idea to check that case.